lib/tjbot.js

```diff
@@ -199,8 +199,23 @@ TJBot.prototype._setupCamera = function() {
         encoding: 'jpg',
         outputDir: './',
         verticalFlip: this.configuration.see.camera.verticalFlip,
-        horizontalFlip: this.configuration.see.camera.horizontalFlip
+        horizontalFlip: this.configuration.see.camera.horizontalFlip,
+        time: 1
```

Comment:
Would be great if you can confirm the `t=1` flag solves the earlier 6 seconds problem I mentioned earlier. 
Note: benchmarchs using spawn takes about 1.2 seconds.